lang/en.json

```
@@ -1633,7 +1633,7 @@
   "pledge.contactToClaim": "To claim this Collective, contact <SupportLink></SupportLink>.",
   "pledge.definition": "A pledge is a way to show interest in supporting a cause or project that is not yet on Open Collective, just like {collective}. If they create a Collective, you will receive an email asking you to fulfill your pledge.",
   "pledge.ownerQuestion": "Do you own {collective}?",
-  "pledge.stats": "{orgCount, plural, =0 {} one {# organization} other {# organizations}} {both, plural, =0 {} one { and }} {userCount, plural, =0 {} one {# individual } other {# individuals }} {totalCount, plural, one {has } other {have }} already pledged a total of {amount} {currency}",
+  "pledge.stats": "{orgCount, plural, =0 {} one {# organization} other {# organizations}} {both, plural, =0 {} one { and } other {}} {userCount, plural, =0 {} one {# individual } other {# individuals }} {totalCount, plural, one {has } other {have }} already pledged a total of {amount} {currency}",
```

If it works here also

```suggestion
  "pledge.stats": "{orgCount, plural, =0 {} one {# organization} other {# organizations}} {both, plural, =0 {} other { and }} {userCount, plural, =0 {} one {# individual } other {# individuals }} {totalCount, plural, one {has } other {have }} already pledged a total of {amount} {currency}",
```

---

Yes this works. 😄